charts/csi-baremetal-deployment/templates/csi-baremetal_v1_deployment.yaml

```diff
@@ -2,13 +2,13 @@ apiVersion: csi-baremetal.dell.com/v1
 kind: Deployment
 metadata:
   name: {{ .Release.Name }}
-  namespace: {{ .Release.Namespace }}
   labels:
     app.kubernetes.io/name: csi-baremetal
     app.kubernetes.io/managed-by: csi-baremetal-operator
     app.kubernetes.io/version: 1.0.0
 spec:
   platform: {{ .Values.platform }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Why do we need namespace in spec?

Reply:
We created  csi-deployment CRD as cluster-scoped, so namespace in meta section was empty ever (even with set value). And if we want to create controlled resources in a custom namespace, we should pass this parameter another way

Reply:
Shouldn't csi-deployment be namespaced? Correct me if I'm wrong

Reply:
Csi-deployment scope was changed to namespaced